ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/efa-pg/provision/cucushift-installer-rehearse-aws-ipi-efa-pg-provision-chain.yaml

```diff
@@ -0,0 +1,22 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-efa-pg-provision
+  steps:
+  - chain: ipi-conf-aws
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  env:
+  - name: CONTROL_PLANE_INSTANCE_TYPE
+    default: "m6i.xlarge"
+    documentation: "Instance type for control plane nodes"
+  - name: COMPUTE_NODE_TYPE
+    default: "m5.xlarge"
+    documentation: "Instance type for compute nodes"
+  - name: ENABLE_EFA_PG_INSTANCE
+    default: "true"
+    documentation: "Enable one compute node using c5n.9xlarge instance type and EFA network interface type in pgcluster placement group"
+  - name: ZONES_COUNT
+    default: "3"
+  - name: ADD_ZONES
```

Comment:
The install workflow does not need `ZONES_COUNT` and `ADD_ZONES`, right?

Reply:
Seems `ADD_ZONES` doesn't need, let me remove it, but if `ZONES_COUNT` doesn't set, it will only create two machinesets in two zones by default, so maybe it will create two such big instances.

Reply:
ah, you are right.